gtsfm/multi_view_optimizer.py

```diff
@@ -24,6 +24,8 @@
 from gtsfm.bundle.bundle_adjustment import BundleAdjustmentOptimizer
 from gtsfm.data_association.data_assoc import DataAssociation
 
+# Paths to Save Output in React Folders.
+REACT_METRICS_PATH = "rtf_vis_tool/src/result_metrics"
```

Comment:
Thanks for the PR, @adisingh50.

this should be relative to the file's location:
```
REACT_METRICS_PATH = Path(__file__).resolve().parent.parent / "rtf_vis_tool" / "src" / "result_metrics"
```

Reply:
Got it. I'll make this edit for both React file paths.